tools/testbuild.sh

```diff
@@ -196,12 +201,21 @@ function build {
 # Coordinate the steps for the next build test
 
 function dotest {
-  echo "===================================================================================="
+  if [ ${PRINTLISTONLY} -ne 1 ]; then
+    echo "===================================================================================="
+  fi
   config=`echo $1 | cut -d',' -f1`
   re="-${config/\//:}[[:space:]]"
   if [[ "${blacklist} " =~ $re ]]; then
+    if [ ${PRINTLISTONLY} -eq 1 ]; then
+      return
+    fi
     echo "Skipping: $1"
   else
+    if [ ${PRINTLISTONLY} -eq 1 ]; then
```

Comment:
Move after line 220?

Reply:
why?

Reply:
Line 219 already print the config name, it doesn't make sense to dup the code here.

Reply:
i wanted machine-readable output. maybe it's a matter of preference.

Reply:
i reverted the output changes to keep the change minimum. i guess you prefer this version.